pnpm-workspace.yaml

```diff
@@ -38,7 +38,7 @@ catalog:
   '@types/react-dom': '^19.1.6'
   '@vanilla-extract/vite-plugin': ^5.1.1
   '@vocab/vite': ^0.2.3
-  braid-design-system: '^33.11.0'
+  braid-design-system: '33.11.0'
```

Comment:
We intentionally keep this unlocked so lockfile updates can get new versions. I think if you unpin this now it _should_ stay resolved to `33.11.0` though.